.changeset/kind-singers-roll.md

```diff
@@ -0,0 +1,6 @@
+---
+'@interledger/open-payments': major
+---
+
+- `createAuthenticatedClient` can now also load a key using a path to the private key file as an argument to `privateKey`
+- `walletAddress` is required in the incoming payment creation request
```

Comment:
Unfourtunately had to make this a major bump because of the `walletAddress` being required now